templates/taxi/manufacturer_list.html

```diff
@@ -21,30 +24,30 @@ <h1>
       {% for manufacturer in manufacturer_list %}
         <tr>
           <td>
-              {{ manufacturer.id }}
+            {{ manufacturer.id }}
           </td>
           <td>
-              {{ manufacturer.name }}
+            {{ manufacturer.name }}
           </td>
           <td>
-              {{ manufacturer.country }}
+            {{ manufacturer.country }}
           </td>
           <td>
-              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
-                Update
-              </a>
-            </td>
-            <td>
-              <a style="color: red"
-                href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">
-                Delete
-              </a>
-            </td>
+            <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
+              Update
+            </a>
+          </td>
+          <td>
+            <a style="color: red"
```

Comment:
Consider using a CSS class instead of inline styling for the 'color: red' property. This will make the code cleaner and easier to maintain.